_posts/2020-08-24-training-imagenet-with-r/training-imagenet-with-r.Rmd

```diff
@@ -0,0 +1,270 @@
+---
+title: "Training ImageNet with R"
+description: >
+  This post explores how to train large datasets with TensorFlow and R. Specifically, we present how to download and repartition ImageNet, followed by training ImageNet across multiple GPUs in distributed environments using TensorFlow and Apache Spark.
+author:
+  - name: Javier Luraschi
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+date: 08-24-2020
+categories:
+  - R
+  - TensorFlow/Keras
+  - Distributed Computing
+  - Data Management
+bibliography: bibliography.bib
+preview: images/fishing-net.jpg
+output:
+  distill::distill_article:
+    self_contained: false
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE)
+```
+
+[ImageNet](http://www.image-net.org/) [@deng2009imagenet] is an image database organized according to the [WordNet](http://wordnet.princeton.edu/) [@miller1995wordnet] hierarchy which, historically, has been used in computer vision benchmarks and research. However, it was not until AlexNet [@krizhevsky2012imagenet] introduced deep learning concepts like convolutional-neural-networks and GPUs, that the computer vision discipline turned to deep learning to achieve state-of-the-art models that revolutionized their field. Given the importance of ImageNet and AlexNet, this post introduces the tools and techniques that you can consider using when training ImageNet and other large-scale datasets with R.
```

Comment:
```suggestion
[ImageNet](http://www.image-net.org/) [@deng2009imagenet] is an image database organized according to the [WordNet](http://wordnet.princeton.edu/) [@miller1995wordnet] hierarchy which, historically, has been used in computer vision benchmarks and research. However, it was not until AlexNet [@krizhevsky2012imagenet] introduced deep learning concepts like convolutional-neural-networks and GPUs, that the computer-vision discipline turned to deep learning to achieve state-of-the-art models that revolutionized their field. Given the importance of ImageNet and AlexNet, this post introduces the tools and techniques that you can consider using when training ImageNet and other large-scale datasets with R.
```

Reply:
Hyphenated "computer-vision" since it's modifying discipline. Approve of @skeydan's change, too, there's just no way to suggest on a suggestion.